images/echoserver/echoserver.go

```diff
@@ -33,6 +37,14 @@ type RequestAssertions struct {
 	Headers map[string][]string `json:"headers"`
 
 	Context `json:",inline"`
+
+	TLSAssertions *TLSAssertions `json:"tls,omitempty"`
+}
+
+// TLSAssertions contains information about the TLS connection
+type TLSAssertions struct {
```

Comment:
WDYT about adding `ServerName`, `NegotiatedProtocol` and `CipherSuite` as well?

Reply:
Good suggestion. I've added the fields.